packages/wrangler/src/is-ci.ts

```diff
@@ -9,6 +9,8 @@ import isCI from "is-ci";
 export const CI = {
 	/** Is Wrangler currently running in a CI? */
 	isCI() {
-		return isCI;
+		return (
+			isCI || process.env.CF_PAGES === "1" || process.env.WORKERS_CI === "1"
```

Comment:
https://github.com/cloudflare/workers-sdk/pull/7549/files#r1886684817

like here for example